src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+
+  if (days >= 7) {
+    return days * rentPrice - sevenDaysDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days >= 3) {
```
You have already done the check for 7 days above

Reply:
Great job. There is a small recommendation that does not affect the correctness of the program.

Reply:
agree